vscode-antimony/all-requirements.txt

```diff
@@ -5,7 +5,7 @@ colorama==0.4.4
 colorlog==4.8.0
 easydev==0.11.0
 idna==2.10
-git+git://github.com/lark-parser/lark.git@5b8c04ca83b9#egg=lark_parser
+#git+git://github.com/lark-parser/lark.git@5b8c04ca83b9#egg=lark_parser

```

Comment:
Any reason why you commented on this out? I fixed the install problem and it should work now.